Makefile

```diff
@@ -90,6 +98,8 @@ up.full: require create-if-missing.env.full wordpress/.env tmp-downloads/owid_ch
 
 	@echo '==> Building grapher'
 	yarn install
+	npx lerna bootstrap
+	npx lerna run build
```

Comment:
the first time `lerna run build` runs, it fails. As I understood it, all builds run in parallel, which means that e.g. utils is not compiled by the time grapher or core-table needs it.

Reply:
This shouldn't be the case. Nx understands the build's dependencies and sequences them appropriately.

I tried to reproduce this failure:
1. `git checkout master` 
2. `rm -rf packages node_modules itsJustJavascript`
3. `git checkout lerna`
4. `make up.full`

And it spun up without complaint, building utils, then core-table, then grapher.

Could you please try to reproduce the failure? 🙏

Reply:
@ikesau sending you a screen grab